libs/components/indicators/src/lib/modules/label/fixtures/label.component.fixture.ts

```diff
@@ -1,11 +1,15 @@
-import { Component } from '@angular/core';
+import { Component, ViewChild } from '@angular/core';
 
 import { SkyLabelType } from '../label-type';
+import { SkyLabelComponent } from '../label.component';
 
 @Component({
   selector: 'sky-test-cmp',
   templateUrl: './label.component.fixture.html',
 })
 export class LabelTestComponent {
-  public labelType: SkyLabelType = 'info';
+  @ViewChild('labelCmp')
+  public labelCmp: SkyLabelComponent;
```

Comment:
Where is this property referenced? Searching for `labelCmp` in the PR didn't turn up any results other than here and in the template.

Reply:
Good catch. That was from an earlier iteration and I missed removing it. Removed.